Document/0x05d-Testing-Data-Storage.md

```diff
@@ -796,6 +796,11 @@ After executing all available app functions, attempt to back up via `adb`. If th
 ```shell
 $ adb backup -apk -nosystem <package-name>
 ```
+If the afore adb backup command gives the message  "Now unlock your device and confirm the backup operation", but the mobile phone does not ask for the password to encrypt the package. The mobile phone should promt the Full Backup windows and asks for a password, this password later used for tar extraction. If the phone does not promt, try afore command with qoutes:
```

Comment:
I rephrased and restructured the sentence:
ADB should respond now with "Now unlock your device and confirm the backup operation" and you should be asked on the Android phone for a password. This is an optional step and you don't need to provide one. If the phone does not prompt this message, try the following command including the quotes:

Please double check if the meaning is still correct. Thanks

Reply:
The meaning still remains. Thanks for the help.

Reply:
Ok. Can you then please update the PR so we can merge? Thanks

Reply:
updated! @sushi2k

Reply:
Can you please also add this, as you also have a source and know the reason why this happens:

The problem happens when your device has an adb version prior to 1.0.31. If that's the case you must use an adb version of 1.0.31 also on hour host machine. Versions of adb after 1.0.32 [broke the backwards compatibility](https://issuetracker.google.com/issues/37096097 "adb backup is broken since ADB version 1.0.32").

Reply:
@sushi2k , I added, can you check?